File: SEM_5/Python/Assignment-7/Q2.py
def are_anagrams(str1, str2):
    str1, str2 = str1.replace(" ", "").lower(), str2.replace(" ", "").lower()
    return sorted(str1) == sorted(str2)
print(are_anagrams("listen", "silent"))
print(are_anagrams("hello", "world"))

File: SEM_1 (ICP)/Output Questions 2/Q2.java
public class Q2 {
    public static void main(String[] args) {
        boolean i=false,x=false,y=true;
        if(i=(x=i=y)) 
            System.out.println("bye");
        else
            System.out.println("hello");
        System.out.println("hi");
    }
}
